RFXtrx/lowlevel.py

```diff
@@ -1600,7 +1601,11 @@ def load_receive(self, data):
         self.id1 = data[4]
         self.id2 = data[5]
         self.rainrate1 = data[6]
+        if self.subtype == 9:
+            self.rainrate1 = data[9]
         self.rainrate2 = data[7]
+        if self.subtype == 9:
+            self.rainrate2 = data[10]
```

Comment:
This is wrong. These types don't have rainrate at all according to docs.

Reply:
thanks for the info.
It's correct there is only a flip counter 
I didn't want to change too much as I am not a programmer but I will start with a new test file and try to recreate all.

the flip counter continues to max and restarts from 0. the rain rate was to calculated as the rainrate increases with .254 mm per tip as indicated in the documentation
you are correct that all accumulations are done in the receiver part thus in the application afterwards.